config-rules.tf

```diff
@@ -295,6 +299,51 @@ resource "aws_config_config_rule" "required-tags" {
     owner             = "AWS"
     source_identifier = "REQUIRED_TAGS"
   }
+}
+
+
+#
+# The rules that follow have been taken from the following module:
+#  https://github.com/osodevops/aws-terraform-module-config-rules/blob/master/aws_config_rules.tf
+#
+
+data "template_file" "aws_config_ami_approved_tag" {
+  template = "${file("${path.module}/config-policies/ami-approved-tag.tpl")}"
+
+  vars = {
+    ami_required_tag_key_value = var.ami_required_tag_key_value
+  }
+}
+
+#==================================================
+# 0.1	Checks whether running instances are using specified AMIs.
```

Comment:
What is 0.1 referring too?

Reply:
We'll clean up the unnecessary comments here.